salt/pillar/makostack.py

```diff
@@ -50,214 +53,221 @@
 .. code:: yaml
 
     ext_pillar:
-      - makostack:
-          - /path/to/stack1.cfg
-          - /path/to/stack2.cfg
+    - makostack:
+      - /path/to/infrastructure.cfg
+      - /path/to/production.cfg
 
 Select config files through grains|pillar|opts matching
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You can also opt for a much more flexible configuration: MakoStack allows one to
+You can also use a much more flexible configuration:  MakoStack allows one to
 select the config files for the current minion based on matching values from
-either grains, or pillar, or opts objects.
+either grains, or pillar, or opts.
```

Comment:
grains, pillar, or opts

Reply:
Still need to fix this

Reply:
OOooo. I was looking for "opts objects" lol.